core/src/main/java/org/mule/runtime/core/internal/context/FlowStoppedPersistenceListener.java

```diff
@@ -9,6 +9,8 @@
 
 /**
  * Defines a listener to persist stop events of Mule flows.
+ *
+ * @since 4.2
```

Comment:
@since 4.4.0 4.3.1 4.2.3 (where this will be cherrypicked, the next patch release)